app/models/user.rb

```diff
@@ -1,6 +1,8 @@
 class User < ActiveRecord::Base
   include AndelaValidator
   include ActionView::Helpers::DateHelper
+  include NewNotification
+
```

Comment:
`NewNotification` sports a very active **after_save**; including it here should cause it to trigger whenever users are saved. I don't think that's desirable or is there other use cases I'm missing?